fix-code-1/main.cpp

```diff
@@ -6,11 +6,11 @@ using namespace std;
 int main() {
     int DAYS_PER_YEAR = 365;
     float average_rainfall = .23;
-    float AverageSnowfall = .12;
+    float Average_Snowfall = .12;
 
-    float averagePrecipitation = average_rainfall + AverageSnowfall;
+    float average_Precipitation = average_rainfall + Average_Snowfall;
```

Comment:
lower snake case variable name. refer to lesson

Reply:
Done.